### DISCUSSION TITLE
[SOLVED] No todas los elementos/propiedades aparecen en un selector en un formulario.

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#115] [4. Bug Reports] [answered]
Hola

Resulta que en el survey que usamos para crear tareas a los proyectos, usamos un componente selección múltiple de tipo colección  llamado "Clientes". El problema es que cuando quiero seleccionar el cliente "A" en el formulario, no me lo muestra en la lista. Se revisó dentro de la colección y se encuentra activo. Se pensó que hubiera sido los schemaInstance, pero probamos con otro cliente y se puede ver.

Adjunto video mostrando lo indicado.
**Link:** https://drive.google.com/file/d/1VC4clVUhxfyNvlqsMfjq40O0451e0WO9/view?usp=sharing

Actualmente cuento con la versión web Versión 15.6.1 (108) y en la versión 15.5.5 funcionaba (se consultó alguien del equipo).

Gracias :)

**Edit:** Una idea que puede ser el incoveniente, es que ese cliente tenía tareas asociadas a la propiedad, y cuando se resolvieron (closed), nunca más apareció el cliente en la lista. Quizas puede ser que mientras una colección se encuentra en una tasks (en los campos adicionales status), puede visualizarse.

**Comment by acedward** [comment, marked as answer]
Es un bug. Será corregido a la brevedad por su gravedad.

**Reply by NicolasPLG** [reply]
gracias @acedward

**Comment by agallardol** [comment]
Por su gravedad lo corregimos hoy mismo. ya está disponible en nuestras aplicaciones!

**Reply by NicolasPLG** [reply]
gracias @agallardol
